### PR TITLE
Logging Change Proposal, Bug Fix for VxAnalyzer, and Fail Faster in Netpool Analysis

### PR DESCRIPTION
**Pull request by svidovich**
Hello! I am back with another PR. I hope that it meets you well!

This PR fixes an exception in `analyze_active_task` in VxAnalyzer. When `active_qhead` is `None`, the function causes an exception. I avoid this by nesting all of that code in an `if active_qhead` block.

This PR also proposes a change in logging, and if you enjoy the change, I would gladly propagate it throughout the rest of the project. Loggers, in my opinion, should not be a part of the object. Instead, just instantiate a common logger in a script, and have a class use that. Mere suggestion -- if you prefer the way you have it, I will gladly revert.

Finally, I noticed that Netpool analysis could fail faster: since it is unsupported in VxWorks 6.x, we should ask for the VxWorks version immediately, and return immediately supposing that 6.x is chosen.

### CODE REVIEW

File: firmware_tools/ghidra/vxhunter_analysis.py [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 from vxhunter_core import *
 from vxhunter_utility.function_analyzer import *
 from vxhunter_utility.symbol import *
 from vxhunter_utility.common import create_initialized_block
 from ghidra.program.model.symbol import RefType, SourceType
-import json
 
 
+
+logger = logging.getLogger(__name__)
```
Comment (dark-lbp) [inline firmware_tools/ghidra/vxhunter_analysis.py]: Please don't move the logger, it's make VxAnalyzer support using customized logger.

**Comment by svidovich** [comment]
Yes -- I'll have it back in soon!

**Comment by svidovich** [comment]
Okay -- I've got the logging the way it was before -- please check it out and let me know if you're happy!!

File: firmware_tools/ghidra/vxhunter_analysis.py [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 from vxhunter_utility.function_analyzer import *
 from vxhunter_utility.symbol import *
 from vxhunter_utility.common import create_initialized_block
 from ghidra.program.model.symbol import RefType, SourceType
-import json
 
 
 class VxAnalyzer(object):
-    def __init__(self, logger=None):
+    def __init__(self):
```
Comment (dark-lbp) [inline firmware_tools/ghidra/vxhunter_analysis.py]: missing `logger=None`

**Comment by dark-lbp** [comment]
There still some codes need to revert and this will make commit history confusion. I will close this PR, please open a new PR from 340c2bf0df0637123960cc610b3199ac5a64430a.

**Pull request closed without merging**